packages/core/src/browser/frontend-application-module.ts

```diff
@@ -53,6 +54,8 @@ export const frontendApplicationModule = new ContainerModule((bind, unbind, isBo
     bindContributionProvider(bind, OpenHandler);
     bind(DefaultOpenerService).toSelf().inSingletonScope();
     bind(OpenerService).toDynamicValue(context => context.container.get(DefaultOpenerService));
+    bind(HttpOpenHandler).toSelf().inSingletonScope();
+    bind(OpenHandler).toDynamicValue(ctx => ctx.container.get(HttpOpenHandler)).inSingletonScope();
```

Comment:
Just a remark  and not a good enough reason for another round :)
If there is no client for `HttpOpenHandler` specifically, a simple binding like below is sufficient:
```
bind(OpenHandler).to(HttpOpenHandler).inSingletonScope();
```